Modules/QuestieShutUp.lua

```diff
@@ -3,6 +3,44 @@ local QuestieShutUp = QuestieLoader:CreateModule("QuestieShutUp")
 ---@type l10n
 local l10n = QuestieLoader:ImportModule("l10n")
 
+-- Compatibility: 2.5.5+ uses ChatFrameUtil.AddMessageEventFilter/RemoveMessageEventFilter instead of ChatFrame_AddMessageEventFilter/RemoveMessageEventFilter
+local ChatFrameAddMessageEventFilter = ChatFrameUtil and ChatFrameUtil.AddMessageEventFilter or ChatFrame_AddMessageEventFilter
+local ChatFrameRemoveMessageEventFilter = ChatFrameUtil and ChatFrameUtil.RemoveMessageEventFilter or ChatFrame_RemoveMessageEventFilter
```

Comment:
Good catch that we also need this change here!

I am thinking about moving this to the [QuestieCompat](https://github.com/Questie/Questie/blob/master/Modules/QuestieCompat.lua) module, so we have it at a single place. Otherwise I can promise I forget about this place if I change something at the other...